freetextresponse/freetextresponse.py

```diff
@@ -564,6 +592,12 @@ def submit(self, data, suffix=''):
             # even if word count is invalid.
             self.count_attempts += 1
             self._compute_score()
+            # check if instructor wants to display other responses
```

Comment:
Since the `if` check has evolved, we may want to just drop this comment instead of worrying about keeping it in sync. The `if` conditions are pretty clear already.